test-network-function/networking/suite.go

```diff
@@ -324,3 +348,111 @@ func testNodePort(env *config.TestEnvironment) {
 		}
 	})
 }
+
+func parseVariables(res string, declaredPorts map[key]string) error {
+	var p Port
+	err := json.Unmarshal([]byte(res), &p)
+	if err != nil {
+		return err
+	}
+	for element := range p {
+		var k key
+		k.port = p[element].ContainerPort
+		k.protocol = p[element].Protocol
+		declaredPorts[k] = p[element].Name
+	}
+	return nil
+}
+func declaredPortList(container int, podName, podNamespace string, declaredPorts map[key]string) error {
+	ocCommandToExecute := fmt.Sprintf(commandportdeclared, podName, podNamespace, container)
+	res, _ := utils.ExecuteCommand(ocCommandToExecute, ocCommandTimeOut, interactive.GetContext(false))
+	err := parseVariables(res, declaredPorts)
+	if err != nil {
+		log.Error(err)
+	}
+	return err
+}
+
+func listeningPortList(commandlisten []string, nodeOc *interactive.Context, listeningPorts map[key]string) error {
+	var k key
+	listeningPortCommand := strings.Join(commandlisten, " ")
+	fmt.Println(listeningPortCommand)
```

Comment:
debug log?

Reply:
I was delete this line : fmt.Println(listeningPortCommand)